server/config/config.go

```diff
@@ -1102,6 +1104,8 @@ type PDServerConfig struct {
 	TraceRegionFlow bool `toml:"trace-region-flow" json:"trace-region-flow,string,omitempty"`
 	// FlowRoundByDigit used to discretization processing flow information.
 	FlowRoundByDigit int `toml:"flow-round-by-digit" json:"flow-round-by-digit"`
+	// UseBucket is the switch to use bucket.
+	UseBucket bool `toml:"use-bucket" json:"use-bucket,string"`
```

Comment:
Better use `EnableRegionBuckets`,`DisableRegionBuckets` or `EnableDynamicRegion`,`DisableDynamicRegion`. CC @BusyJay

Reply:
`EnableXXX` is better.

Reply:
fix